packages/slice-machine/components/KebabMenuDropdown/KebabMenuList.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { Box, Flex } from "theme-ui";
+import { MenuOption } from ".";
+
+const MenuItem: React.FunctionComponent<{
+  value: string;
+  onClick: (event: React.MouseEvent<HTMLDivElement>) => void;
+}> = ({ value, onClick }) => {
+  return (
+    <Box
+      sx={{
+        p: 2,
+        cursor: "pointer",
+        fontSize: "14px",
+        borderRadius: 0,
+        ":hover": {
+          bg: "hoverBackground",
+        },
+      }}
+      onClick={onClick}
+    >
+      {value}
+    </Box>
+  );
+};
+
+export const KebabMenuList: React.FunctionComponent<{
```

Comment:
really thought the Kebab naming was a troll 😅  until I found this
![image](https://user-images.githubusercontent.com/41187899/199735500-ee0af102-a999-4122-be44-12888e4d9e58.png)

Reply:
I also couldn't believe it 😂 

There is also a doner menu
<img width="82" alt="image" src="https://user-images.githubusercontent.com/47107427/199744149-ba624d9d-f867-47f8-b42b-03989bb485f1.png">